github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```diff
@@ -77,6 +77,7 @@ orgs:
       - stu-gott
       - tiraboschi
       - vasiliy-ul
+      - vamsikrishna-siddu
```

Comment:
I think your username should be added ahead of `vasiliy-ul` for it to be sorted alphabetically.

Reply:
done